packages/design-system-react/src/components/AvatarBase/AvatarBase.tsx

```diff
@@ -1,5 +1,5 @@
 import { Slot } from '@radix-ui/react-slot';
-import React from 'react';
+import React, { forwardRef } from 'react';
```

Comment:
This has been moved to a separate PR https://github.com/MetaMask/metamask-design-system/pull/673